src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -369,8 +352,11 @@ public function get_prices( \WC_Product $product, $tax_display_mode = '', $qty =
 		$tax_display_mode = in_array( $tax_display_mode, [ 'incl', 'excl' ], true ) ? $tax_display_mode : get_option( 'woocommerce_tax_display_shop' );
 		$price_function   = 'incl' === $tax_display_mode ? 'wc_get_price_including_tax' : 'wc_get_price_excluding_tax';
 
-		// Get individual product prices from product schema.
-		$prices = $this->product_schema->get_prices( $product, $tax_display_mode );
+		// Get individual product prices.
+		$prices['price']         = $this->prepare_money_response( $price_function( $product ), wc_get_price_decimals() );
```

Comment:
It's not completely the same because of the raw prices needed here. It didn't seem worth constructing the other class to just take a few lines of logic when this class already has the $product object and all other values.